### ISSUE TITLE
Modifying an AHK version to track time spent in any app

### ISSUE THREAD

**Issue by HuanNV9** [RustingSword/ticker#1]
Hello, the mobile version works for any app and has two timers:
A timer that counts the time spent in the current session (displayed at the top)
A timer that counts the time spent in the day (displayed at the bottom)
These timers are always visible and work for any app.
However, the AHK version, as I understand it, only works with the PDF reader. How can I modify it to function like the mobile version and work with any app?
I am also experiencing a font error as shown in the image below.

<img width="119" alt="font error" src="https://user-images.githubusercontent.com/91595189/230659402-c8638300-80d7-4eae-9275-519ae534642a.PNG">

**Comment by RustingSword** [comment]
Sorry I'm not sure what you mean by mobile version? As for the AHK script, you can add corresponding code like `WinActive("ahk_class classFoxitReader")` (change `classFoxitReader` to the class of your preferred application) to record time spent on that application.

Please make sure you are not using any non-ascii characters in the script to see whether there's still garbled text.

**Comment by ooker777** [comment]
after those time I'm surprised that you can still remember what the code does

**Comment by RustingSword** [comment]
It's only a few lines of code after all :-)
I find that it still works greatly for passive time tracking, and I'm considering to find something similar for Linux/macOS since I'm not using Windows anymore.

**Comment by ooker777** [comment]
I guess python?

**Comment by RustingSword** [comment]
Yes, python would be my first choice. After some quick search and gpt chatting, it seems straightforward to get active window name/title, but a bit tricky to get urls from browser without installing some extensions. Anyways, It's definitely doable, and I will try to hack something out.

**Comment by HuanNV9** [comment]
Hi, 
I'm really sorry for the delayed response. Somehow, I missed your message earlier.
Regarding the mobile app, I was referring to a Tasker app created by Ooker777. You can find more information about it on this Reddit thread:
https://www.reddit.com/r/tasker/comments/bbw8zx/share_corner_timer_displaying_app_usage_in_screen/
You're right that I can use a class to record the time spent on applications, but I think it's too broad as there are many apps with the same class. What I really want to do is track the time spent on every application on my PC, just like the mobile version does.
As for the font error, I wasn't able to identify the problem. I didn't make any modifications to your code, but the font error still persists. I'm not sure if it's related to my PC's font settings or not.

**Comment by RustingSword** [comment]
I see, then you can change `If (WinActive(...))` to [`WinGet`](https://www.autohotkey.com/docs/v1/lib/WinGet.htm). More specifically, use `WinGet, OutVar, ProcessName, A` to save the process name of the active window in variable `OurVar`, and store time usage by each application in the log file.
